skyportal/handlers/api/group.py

```diff
@@ -440,7 +445,11 @@ def post(self, group_id, *ignored_args):
                 f"User {user_id} is already a member of group {group_id}."
             )
 
-        DBSession().add(GroupUser(group_id=group_id, user_id=user_id, admin=admin))
+        DBSession().add(
+            GroupUser(
+                group_id=group_id, user_id=user_id, admin=admin, can_save=can_save
```

Comment:
We're not checking if `can_save` is indeed a Boolean so this will likely throw an error if the API is called with `t` or the like.